README.md

```diff
@@ -23,6 +23,9 @@
 - [Sponsors](#sponsors)
 - [Support](#support) ([documentation](https://json.nlohmann.me), [FAQ](https://json.nlohmann.me/home/faq/), [discussions](https://github.com/nlohmann/json/discussions), [API](https://json.nlohmann.me/api/basic_json/), [bug issues](https://github.com/nlohmann/json/issues))
 - [Examples](#examples)
+  - [Read JSON from a file](#read-json-from-a-file)
+  - [Creating `json` objects from JSON literals](#creating-json-objects-from-json-literals)
+  - [Reading and writing individual object keys](#reading-and-writing-individual-object-keys)
   - [JSON as first-class data type](#json-as-first-class-data-type)
```

Comment:
This section has disappeared.

Reply:
good catch, thank you!

Reply:
I think it was good having a section titled "JSON as first-class data type" - so instead of removing it altogether I think it should remain.

Reply:
I have put the old section back. If you have a moment, I'd love another review.